hil/api.py

```diff
@@ -1248,7 +1248,9 @@ def show_headnode(nodename):
     Example:  '{"name": "headnode1",
                 "project": "project1",
                 "hnics": ["hnic1", "hnic2"],
-                "vncport": 5900
+                "vncport": 5900,
+                "uuid": <headnode uuid>, 
+                "base_img": <headnode base_img>
```

Comment:
I think a while back we decided to remove a bunch of this info from the docstrings, in favor of keeping the detailed api descriptions in rest_api.md. Some stuff in this file still has examples because of legacy, but I think the thing to do here is just delete this, rather than try to keep it in sync with the same info elsewhere.

Reply:
Done